src/App.jsx

```diff
@@ -12,7 +12,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Homepage />} />
         <Route path="/allArticles" element={<AllArticles />} />
-        <Route path="/article/:topic" element={<ArticlesByTopic />} />
+        <Route path="/articles/:topic" element={<ArticlesByTopic />} />
```

Comment:
Are allArticles and articles by topic supposed to look different? Do you want to display the all articles vs specific articles in a different way?